spec/studentManager.spec.js

```diff
@@ -0,0 +1,28 @@
+import StudentManager from '../src/studentManager.js'
+import Student from '../src/student.js'
+
+describe('StudentManager:', () => {
+  let studentManager
+
+  beforeEach(() => {
+    studentManager = new StudentManager()
+    Student.resetId()
+  })
+
+  it('create Student', () => {
+    const input = {
+      firstName: 'Bob',
+      lastName: 'Builder',
+      gitHubUserName: 'BobbyBuilder',
+      email: 'bob@builder.com',
+      cohortId: 1
+    }
+
+    const result = studentManager.createStudent(input)
+
+    const expected = { ...input, studentId: 1 }
+
+    expect(result).toBe(1)
```

Comment:
brb gotta check that .toBe() method and probably steal this as a practice

Reply:
The way I'm using it here isn't particularly too useful over toEqual here tbh, handy to know it exists though!